### PR TITLE
fix scrollbars in ie11

### PR DESCRIPTION
**Pull request by ocke**
/cc @zendesk/vegemite 

before
screen shot 2018-03-28 at 11 03 00

after
screen shot 2018-03-28 at 11 03 11

### CODE REVIEW

File: assets/iframe.html [hunk 1]
```diff
@@ -5,6 +5,11 @@
   <script type="text/javascript" src="https://assets.zendesk.com/apps/sdk/2.0/zaf_sdk.js"></script>
   <link rel="stylesheet" href="https://assets.zendesk.com/apps/sdk-assets/css/2/zendesk_garden.css" type="text/css">
   <script type="text/javascript" src="main.js"></script>
+  <style>
+    html { overflow: hidden; }
+    body { overflow: hidden; }
+    div[data-main] { overflow: auto; }
+  </style>
 </head>
```
Comment (vaevictis) [inline assets/iframe.html]: I'd suggest it's time to move that stuff in, _ya'now_, a dedicated `CSS` file.

Comment (ocke) [reply assets/iframe.html]: I disagree, but i don't use the text app, so who cares if thousands of agent need to wait a bit longer for their app to work.

**Comment by sostopher** [comment]
This doesn't appear to fix the issue that the customer complained about, which is the bottom scrollbar obscuring content. Unless I'm mistaken?

**Comment by ocke** [comment]
you are mistaken. in the before there is text under the horizontal scrollbar

**Pull request merged**